samples/tls/logging/README.md

```diff
@@ -37,9 +24,26 @@ my_repository_path/logging/bin/deploy_logging_open.sh
 ```
 ## Notes On Customization Values
 
-Set `MON_TLS_ENABLE=true` in the `$USER_DIR/logging/user.env` file. This variable modifies the deployment of Open Distro for Elasticsearch to be TLS-enabled.
+### Ingress
+
+* If you are using ingress, set the environment variable `TLS_ENABLED=true` in the `user.env` file in order to use TLS for connections between the user and Kibana.  
+
+* Manually populate these secrets in the `logging` namespace (or `LOG_NS` value) **before**
+you run the deployment script:
+
+  * kubernetes.io/tls secret - `kibana-ingress-tls-secret`
+  * kubernetes.io/tls secret - `elasticsearch-ingress-tls-secret`
+
+Generating these certificates is outside the scope of this example. However, you
+can use the process documented in ["Configure NGINX Ingress TLS for SAS Applications"](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=default&docsetId=calencryptmotion&docsetTarget=n1xdqv1sezyrahn17erzcunxwix9.htm&locale=en#n0oo2yu8440vmzn19g6xhx4kfbrq) in SAS Viya Administration documentation and specify the `logging` namespace.
+
+If any of the required certificates do not exist, the deployment process attempts to use [cert-manager](https://cert-manager.io/) to generate the missing certificates.
+
```

Comment:
Is this true?  I do not believe there is anything in the log deployment scripts that checks for these particular certificates or secrets.  I think the user is required to manually create these secrets (end of story).

Reply:
Yes, the user is required to provide any certificates referenced in the **ingress** section for both monitoring and logging.

Reply:
The **kubectl create secret** command does NOT create TLS certs; it creates Kubernetes secret objects.  Once you have created a TLS cert through some mechanism, you can store them within a Kubernetes secret object to make them available to the deployment scripts and Kubernetes pods.

Do we want to tell people to use cert-manager to create the certs?  Or, should we just tell them they need to create/obtain the TLS certs and then store them in Kubernetes secret objects?  If we mention the kubectl command, we probably need to provide a more complete example of the syntax.  We have some syntax shown in the NOTES_ON_TLS.md file although those are specific to what we need for the Elasticsearch TLS certs -- not sure if the ingress cert secrets are the same.

Reply:
I don't think we should get into how to create the various .crt files - there's CAs and server certs and there are nuances and caveats for each. It's out of scope for this project, imo.

Reply:
I agree that we shouldn't get into the weeds of how to create certs.  But, assuming they have a set of certs, what's the syntax for creating the corresponding Kubernetes secrets?    For the Elasticsearch TLS certs, we don't use Kubernetes TLS secret objects, just Kubernetes generic secrets.  But, I believe the secrets for the ingress certs must be Kubernetes TLS secrets; correct?  If so, is this the syntax (taken from kubectl --help text)?
`kubectl create secret tls NAME --cert=path/to/cert/file --key=path/to/key/file`

Reply:
`kubectl create secret tls "$SECRET_NAME" -n "$NAMESPACE" --key "$CERT_KEY" --cert "$CERT_FILE"`